docs/source/autograd.rst

```diff
@@ -23,6 +23,13 @@ Functional higher level API
     This API is in beta. Even though the function signatures are very unlikely to change, major
     improvements to performances are planned before we consider this stable.
 
+.. note::
+    In the future, we plan to replace the functional higher level API by upstreaming
+    `functorch <https://github.com/pytorch/functorch>`_ after it becomes more mature.
+    functorch is able to compute similar quantities and already has a number of
+    performance improvements but today there are different tradeoffs in which you
```

Comment:
Are these tradeoff discussed anywhere?
I think that if we mention that, we should say how to make that choice.

Reply:
They're not, should I delete this sentence for now?